mohamad-dakdouk/week5/wed/D23-countries-cities-hm/app/models/city.rb

```diff
@@ -0,0 +1,3 @@
+class City < ApplicationRecord
+    belongs_to :country
```

Comment:
once you destroy the country, does the city still exist?
if i click on the city that is connected to the deleted country, it will cause an error. 

try adding either `:dependent => :destroy` here, or doing `:optional => true` to the country model. depends if you think all cities should be destroyed if a country is deleted, or if you think cities can stand independently from countries